frameworks/kafka/src/main/dist/kafka_svc.yml

```diff
@@ -0,0 +1,36 @@
+name: {{FRAMEWORK_NAME}}
```

Comment:
All frameworks have their yaml named `svc.yml`.  Would be nice to stay consistent.